server/routerlicious/kubernetes/routerlicious/templates/fluid-configmap.yaml

```
@@ -136,6 +136,12 @@ data:
         },
         "scribe": {
             "kafkaClientId": "{{ template "scribe.fullname" . }}"
+            "checkpointHeuristics": {
```

@hedasilv Added default values for scribe checkpointing

---

Great! Maybe let's have these values coming from values.yaml? You could add a new section in [values.yaml](https://github.com/microsoft/FluidFramework/blob/d848c292086fadfd27279c3d2f8a8a9f68be5a3f/server/routerlicious/kubernetes/routerlicious/values.yaml#L73) (under scribe):
```suggestion
scribe:
  name: scribe
  replicas: 8
  checkpointHeuristics:
    enable: false
    idleTime: 10000
    maxTime: 60000
    maxMessages: 500
```

And then here, in the fluid-configmap.yaml, you would do:

```suggestion
            "checkpointHeuristics": {
                 "enable": {{ .Values.scribe.checkpointHeuristics.enable }},
                 "idleTime": {{ .Values.scribe.checkpointHeuristics.idleTime }},
                 "maxTime": {{ .Values.scribe.checkpointHeuristics.maxTime }},
                 "maxMessages": {{ .Values.scribe.checkpointHeuristics.maxMessages }}
            }
```

---

Added